WordPress/Classes/ViewRelated/NUX/NUXNoResultsViewController.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+@objc protocol NUXNoResultsViewControllerDelegate {
```

Comment:
Well, I've already had to rename it once as it was too specific initially. 😄  I was trying to be forward-thinking in that it could be used to replace the existing `WPNoResultsView`. Hence the completely generic name. If makes more sense to be more specific, I can rename it, but that was my reasoning.

Reply:
The `NUX` part is the part that isn't generic enough imo as this screen doesn't appear within the `NUX` flows 😜

Reply:
Ah, I see. Gotcha. OK, I'll rename it.